libs/s3/s3_iam_rest_rgw.py

```diff
@@ -238,3 +238,43 @@ async def get_user_info(self,user_params) -> Tuple[HTTPStatus, Dict[str, Any]]:
         status, user_info = await rgwcli.signed_http_request(
             'GET', params.IAM_USER, query_params=user_params)
         return status, user_info
+
+    async def put_user_policy(self,user_params) -> Tuple[HTTPStatus, Dict[str, Any]]:
+        """
+        Illustrate S3Client signed_http_request work.
+        Create IAM user by specifying parameters, HTTP method and path.
+        :returns: HTTP status code and user information as parsed json.
+        """
+
+        rgwcli = S3Client(
+            self.ACCESS_KEY , self.SECRET_KEY, self.HOST, self.PORT, tls_enabled=False)
+        status = await rgwcli.signed_http_request(
+            'POST', "/", query_params=user_params)
+        return status
+
+    async def delete_user_policy(self,user_params) -> Tuple[HTTPStatus, Dict[str, Any]]:
+        """
+        Illustrate S3Client signed_http_request work.
+        Create IAM user by specifying parameters, HTTP method and path.
+        :returns: HTTP status code and user information as parsed json.
+        """
+
+        rgwcli = S3Client(
+            self.ACCESS_KEY , self.SECRET_KEY, self.HOST, self.PORT, tls_enabled=False)
+        status = await rgwcli.signed_http_request(
+            'POST', "/", query_params=user_params)
+        return status
+
+    # pylint: disable=C0103
+    async def delete_policy_with_user_keys(
+        self,user_params,access_key,secret_key) -> Tuple[HTTPStatus, Dict[str, Any]]:
+        """
+        Illustrate S3Client signed_http_request work.
+        Create IAM user by specifying parameters, HTTP method and path.
+        :returns: HTTP status code and user information as parsed json.
+        """
+        rgwcli = S3Client(
+             access_key, secret_key, self.HOST, self.PORT, tls_enabled=False)
+        status = await rgwcli.signed_http_request(
+            'POST', "/", query_params=user_params)
+        return status
```

Comment:
You should merge `delete_user_policy` and `delete_policy_with_user_keys` functions. And replace appropriate calls.
```suggestion
    async def delete_user_policy(self, user_params, access_key=self.ACCESS_KEY,
                                 secret_key=self.SECRET_KEY) -> Tuple[HTTPStatus, Dict[str, Any]]:
        """
        Delete IAM User Policy
        :param user_params: User Parameters.
        :param access_key: Access Key.
        :param secret_key: Secret Key.
        :returns: HTTP status code and user information as parsed json.
        """
        rgw_client = S3Client(access_key, secret_key, self.HOST, self.PORT, tls_enabled=False)
        status = await rgw_client.signed_http_request('POST', "/", query_params=user_params)
        return status
```

Reply:
Will update